aten/src/ATen/native/BatchLinearAlgebraKernel.cpp

```diff
@@ -200,12 +200,10 @@ std::tuple<Tensor, Tensor> eig_kernel_impl(const Tensor& self, bool& eigenvector
                  ? at::empty_strided({n, n}, {1, n}, options)
                  : Tensor();
 
-  // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
-  int64_t info;
+  int64_t info = 0;
```

Comment:
This is not really necessary, as this is used as an output parameter in an external C function. This is also done in other places such as:
https://github.com/pytorch/pytorch/blob/9e4d3378b7b4feec56fcbef4166724f4a48d133b/aten/src/ATen/native/BatchLinearAlgebraKernel.cpp#L159-L162

Reply:
@IvanYashchuk mentioned that this code is part of a deprecated function, so it will be removed in 1.11. As such, changing it or not is not really important.